CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## January 2022
+
+### `1.0.0-beta4` - 02/25/2022
+#### Added
+- .npmignore file
+
+#### Fixed
+- Missing `dist` directory since npm was ignoring it due to lack of .npmignore
+
+### `1.0.0-beta5` - 10/1/2022
```

Comment:
not blocking, but I think changelog should be most recent changes are added to the top

Also, we need to add 

`## October 2022`

tag in style as the beta 4 (which by the way I now see should say February)

in summary, this can be fixed as a fast-follow